services/ui-src/src/components/statusing/ChildRow.test.tsx

```diff
@@ -18,10 +18,12 @@ const mockedUseStore = useStore as jest.MockedFunction<typeof useStore>;
 const childRowComponent = (
   <RouterWrappedComponent>
     <Table>
-      <TableRow page={mockChildRowPage} rowDepth={2} />
-      {mockChildRowPage.children?.map((child) => (
-        <ChildRow key={child.path} page={child} rowDepth={2} />
-      ))}
+      <Tbody>
```

Comment:
Fixes `Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser.`.